galadriel/scenario/state.py

```
@@ -215,20 +242,22 @@ class AddScenarioState(ScenarioState):
 
     form_data:dict = {}
 
-    def handle_submit(self, form_data):
+    async def handle_submit(self, form_data):
         """Validate and create a new scenario from the form."""
         self.form_data = form_data
         result = self.add_scenario(form_data)
 
         if result is None: return rx.toast.error("name cannot be empty")
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)